content/_data/changelogs/lts.yml

```
@@ -12986,6 +12986,493 @@
       Upgrade to Winstone 8.10 and Jetty 12.0.22.
       Jetty upgrade includes multiple bug fixes and most recent release of Eclipse Jetty 12.
 
+- version: "2.516.1"
+  date: 2025-07-23
+  lts_predecessor: "2.504.3"
+  lts_baseline: "2.516"
+  changes: # compared to lts_baseline 2.516 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: regression
+    pull: 10786
+    issue: 75834
+    authors:
+      - scherler
+    pr_title: "[JENKINS-75834] make sure only one instance of the dropdown is visible"
+    message: |-
+      Fixed a bug where multiple dropdowns for the breadcrumb navigation could be visible.
+  - type: major bug
+    category: regression
+    pull: 10791
+    issue: 75832
+    authors:
+      - jtnord
+    pr_title: "[JENKINS-75832] Support actions with Badges in the Hamburger Menu"
+    message: |-
+      Actions with Alerts now display correctly in the new header when in the hamburger menu.
+  - type: major bug
```

I wouldn't class this as a major bug:
```suggestion
  - type: bug
```